src/functions-and-arrays.js

```diff
@@ -78,9 +122,10 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
-
-
+function howManyTimes(arr, word) {
+  if (arr.length === 0) return 0;
+  if (arr.indexOf(word) === -1) return 0;
+}
```

Comment:
For this one, you will need to loop on the array and increment a counter each time you find the target word.

Take a look on the lab solution posted on github.